autogluon/model_zoo/model_zoo.py

```diff
@@ -42,11 +43,12 @@ def get_model(name, **kwargs):
         raise ValueError('%s\n\t%s' % (str(name), '\n\t'.join(sorted(models.keys()))))
     return net
 
+
 def get_model_list():
     """Get the entire list of model names in model_zoo.
     Returns
     -------
     list of str
         Entire list of model names in model_zoo.
     """
-    return models.keys()
+    return list(models.keys()) + list(gcv.model_zoo.get_model_list())
```

Comment:
@zhanghang1989 This appears to be change in functionality, confirm if correct.

Reply:
Thanks @Innixma . This change is good to get gluoncv model_zoo.